pkg/kubelet/cm/cgroupManager_mock.go

```diff
@@ -0,0 +1,45 @@
+// +build linux
```

Comment:
I am having trouble following the discussion here, but I get the sense this is structured wrong.

This should just be in a `/cm/cgroup_manager_test.go` file, and I would think we would have a `../cm/types.go` file.

Still reviewing this PR.